cmd/kfulint/main.go

```diff
@@ -123,11 +113,20 @@ func (l *linter) LoadProgram() {
 }
 
 func (l *linter) InitCheckers() {
-	for _, name := range lint.AvailableCheckers() {
-		// Nil enabledSet means "all checkers are enabled".
-		if l.enabledSet == nil || l.enabledSet[name] {
-			l.checkers = append(l.checkers, checker{name, lint.NewChecker(name, l.ctx)})
+	if l.enabledCheckers == nil {
+		for _, name := range lint.AvailableCheckers() {
+			c, _ := lint.NewChecker(name, l.ctx)
+			l.checkers = append(l.checkers, checker{name, c})
+		}
```

Comment:
Maybe add `return` after this if body?
It's understandable that for loop above will not execute any code in case `l.enabledCherkers == nil`, but
if there ever be other code, things will get ugly.
We should be explicit here.

Reply:
Ok